src/media/js/helpers_local.js

```diff
@@ -153,16 +153,25 @@ define('helpers_local',
         return document.documentElement.dir;
     }
 
+    function getProductType(obj) {
```

Comment:
Maybe we should call `apps.transform()` instead and use `isWebsite`, `isAddon` and `isApp`. I don't know the consequences of calling `apps.transform()` before model caching, though. It *seems* fine because there is a safeguard against double transforming.